### ISSUE TITLE
Crossref dropped support for twitter (X) from 22 February 2023

### ISSUE THREAD

**Issue by dolphin323** [pmongeon/scholars-on-twitter#2]
```
From February 2023 the Twitter agent has been deprecated. As per our contract with Twitter, we have removed all access to Twitter events.
```
https://www.eventdata.crossref.org/guide/sources/twitter/
https://community.crossref.org/t/dropping-twitter-as-an-event-source/3315/2

Maybe someone knows possible replacements?

UPDATE: the Tweeter API endpoint for tweets' search is now paid, therefore there are no free resources now. But it is possible to rewrite the script in this repo using a paid one.

**Comment by pmongeon** [comment]
Yes, that was only one of several unexpected changes that caused issues with the reproducibility of our process (Crossref not collecting X data) and the usability of the dataset (change of OpenAlex author IDs). The process remains unreproducible as it is (although the logic behind it can of course be implemented as long as one has access to disambiguated authorship data and X data). However, we do have a new version of the dataset that is usable. It can be found here: [https://zenodo.org/uploads/10905839](https://zenodo.org/uploads/10905839).
